xl-v1-object.go

```diff
@@ -176,7 +176,18 @@ func (xl xlObjects) renameObject(srcBucket, srcObject, dstBucket, dstObject stri
 		if errCount <= len(xl.storageDisks)-xl.readQuorum {
 			return nil
 		}
-		xl.deleteObject(srcBucket, srcObject)
+		// Rename back the object on disks where RenameFile succeeded
+		for index, disk := range xl.storageDisks {
```

Comment:
["iff"](https://en.wikipedia.org/wiki/If_and_only_if) is not a typo, it expands to "if and only if" in mathematical text. Let me know if I should change it.